src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/resolver.py

```diff
@@ -73,7 +73,16 @@ def fetch_stack_output(self, value, key, optional=False): # pylint: disable=too-
                 self.stage_parameters[parent_key][key] = stack_output
             else:
                 if not stack_output:
-                    raise Exception("No Stack Output found on %s in %s with stack name %s and output key %s" % account_id, region, stack_name, export)
```

Comment:
The formatting of the string got interpreted wrong, resulting in the following error:
```
raise Exception("No Stack Output found on %s in %s with stack name %s and output key %s" % account_id, region, stack_name, export)
TypeError: not enough arguments for format string
```

With this PR this is fixed as well.